cves/CVE-2013-0844.yml

```
@@ -43,11 +43,11 @@ announced_instructions: |
 
 
   Please enter your date in YYYY-MM-DD format.
-announced_date:
+announced_date: 2013/12/07
```

```suggestion
announced_date: 2013-12-07
```
The convention we use on dates is YYYY-MM-DD with dashes.

---

Thanks for catching that